data/json/items/gun/artisan_guns.json

```
@@ -7,20 +7,21 @@
     "description": "A cut-down, modified Mosin-Nagant.  It has a short barrel, and the stock has been cut down and fitted with a mace head.  The sides of the firearm have been reinforced with metal to withstand a beating.",
     "weight": "3000 g",
     "volume": "2220 ml",
-    "longest_side": "40 cm",
+    "longest_side": "35 cm",
+    "//1": "20cm barrel length, 15cm remaining stock.",
     "price": 23000,
     "price_postapoc": 3000,
-    "//": "should be very inaccurate and hard to aim by comparison.  Setting handling to that of pistols",
+    "//2": "should be very inaccurate and hard to aim by comparison.  Setting handling to that of pistols",
     "handling": 10,
     "relative": {
       "range": -30,
-      "ranged_damage": { "damage_type": "bullet", "amount": -6 },
+      "ranged_damage": { "damage_type": "bullet", "amount": -17.3 },
```

I don't think we do fractional damage.
```suggestion
      "ranged_damage": { "damage_type": "bullet", "amount": -17 },
```